### DISCUSSION TITLE
osu! in wine

### DISCUSSION THREAD

**Discussion by rXelelo** [ppy/osu#28091] [Q&A]
so i start osu and have error
os: Monjaro Linux
Intel Pentium G3220
Intel HD Graphics 
    ~/.w/dr/u/r/A/Local/osu!  wine osu\!.exe                    ✔  22s  
02ec:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
02ec:fixme:seh:WerRegisterRuntimeExceptionModule (L"C:\\windows\\Microsoft.NET\\Framework\\v4.0.30319\\mscordacwks.dll", 78F40000) stub
06f0:fixme:actctx:parse_depend_manifests Could not find dependent assembly L"Microsoft.Windows.Common-Controls" (6.0.0.0)
MESA-INTEL: warning: Haswell Vulkan support is incomplete
02ec:fixme:cryptasn:CryptDecodeObjectEx Unsupported decoder for lpszStructType 1.3.6.1.4.1.311.2.1.4
02ec:fixme:cryptasn:CryptDecodeObjectEx Unsupported decoder for lpszStructType 1.3.6.1.4.1.311.2.1.4
02ec:fixme:nls:get_dummy_preferred_ui_language (0x0 0x422 0094E54C 00000000 0094E548) returning a dummy value (current locale)
02ec:fixme:nls:get_dummy_preferred_ui_language (0x0 0x422 0094E54C 0111B448 0094E548) returning a dummy value (current locale)
02ec:fixme:process:SetProcessPriorityBoost (0000030C,0): stub
07a8:fixme:ras:RasEnumConnectionsW (0122EE90,09B7ED90,09B7ED94),stub!
07a8:fixme:ras:RasEnumConnectionsW RAS support is not implemented! Configure program to use LAN connection/winsock instead!
07a8:fixme:ras:RasConnectionNotificationW (FFFFFFFF,000003E0,0x00000003),stub!
02dc:fixme:font:find_matching_face Untranslated charset 255
02dc:fixme:font:get_nearest_charset returning DEFAULT_CHARSET face->fs.fsCsb[0] = 00000000 file = L"\\??\\Z:\\usr\\share\\fonts\\TTF\\MuktiNarrow.ttf"
02dc:fixme:font:find_matching_face Untranslated charset 255
02dc:fixme:font:get_nearest_charset returning DEFAULT_CHARSET face->fs.fsCsb[0] = 00000000 file = L"\\??\\Z:\\usr\\share\\fonts\\TTF\\Pothana2000.ttf"
02dc:fixme:font:find_matching_face Untranslated charset 255
02dc:fixme:font:get_nearest_charset returning DEFAULT_CHARSET face->fs.fsCsb[0] = 00000000 file = L"\\??\\Z:\\usr\\share\\fonts\\TTF\\Sampige.ttf"
02dc:fixme:gdiplus:GdipGetFamilyName No support for handling of multiple languages!
02dc:fixme:gdiplus:GdipCreateHalftonePalette stub
03d8:fixme:virtual:NtFlushProcessWriteBuffers stub
036c:fixme:kernelbase:AppPolicyGetThreadInitializationType FFFFFFFA, 0CA6FF18
03d8:fixme:cryptasn:CryptDecodeObjectEx Unsupported decoder for lpszStructType 1.3.6.1.4.1.311.2.1.4
03d8:fixme:cryptasn:CryptDecodeObjectEx Unsupported decoder for lpszStructType 1.3.6.1.4.1.311.2.1.4
02ec:fixme:actctx:parse_depend_manifests Could not find dependent assembly L"Microsoft.Windows.Common-Controls" (6.0.0.0)
02ec:fixme:nls:GetFileMUIPath stub: 0x10, L"C:\\windows\\system32\\tzres.dll", (null), 0094EF14, 01209EE0, 0094EF18, 0094EF0C
02ec:fixme:nls:GetFileMUIPath stub: 0x10, L"C:\\windows\\system32\\tzres.dll", (null), 0094EF14, 01209EE0, 0094EF18, 0094EF0C
02ec:fixme:nls:GetFileMUIPath stub: 0x10, L"C:\\windows\\system32\\tzres.dll", (null), 0094EF14, 01209EE0, 0094EF18, 0094EF0C
02e0:fixme:crypt:CertAddCertificateLinkToStore (093C3B80, 093BB86C, 00000004, 00000000): semi-stub
02f4:fixme:avrt:AvSetMmThreadCharacteristicsW (L"Audio",0DC0FF30): stub
02ec:fixme:bcrypt:BCryptGetFipsAlgorithmMode 0094CEC0 - semi-stub
02ec:fixme:win:RegisterTouchWindow hwnd 000200D2, flags 0x2 stub!
07a4:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
07a4:fixme:wmiutils:wmiutils_cf_QueryInterface interface {b196b28f-bab4-101a-b69c-00aa00341d07} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {c3fcc19e-a970-11d2-8b5a-00a0c9b7c9c4} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {00000003-0000-0000-c000-000000000046} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {00000144-0000-0000-c000-000000000046} not implemented
07a4:fixme:wmiutils:path_GetInfo some flags are not implemented
07a4:fixme:wmiutils:path_GetInfo some flags are not implemented
07a4:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
07a4:fixme:wmiutils:wmiutils_cf_QueryInterface interface {b196b28f-bab4-101a-b69c-00aa00341d07} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {c3fcc19e-a970-11d2-8b5a-00a0c9b7c9c4} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {00000003-0000-0000-c000-000000000046} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {00000144-0000-0000-c000-000000000046} not implemented
07a4:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
07a4:fixme:wmiutils:wmiutils_cf_QueryInterface interface {b196b28f-bab4-101a-b69c-00aa00341d07} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {c3fcc19e-a970-11d2-8b5a-00a0c9b7c9c4} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {00000003-0000-0000-c000-000000000046} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {00000144-0000-0000-c000-000000000046} not implemented
07a4:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
07a4:fixme:wbemprox:wbemprox_cf_QueryInterface interface {b196b28f-bab4-101a-b69c-00aa00341d07} not implemented
07a4:fixme:wbemprox:wbem_locator_QueryInterface interface {c3fcc19e-a970-11d2-8b5a-00a0c9b7c9c4} not implemented
07a4:fixme:wbemprox:wbem_locator_QueryInterface interface {00000003-0000-0000-c000-000000000046} not implemented
07a4:fixme:wbemprox:wbem_locator_QueryInterface interface {00000144-0000-0000-c000-000000000046} not implemented
07a4:fixme:wbemprox:wbem_locator_ConnectServer authentication not supported
07a4:fixme:wbemprox:wbem_locator_ConnectServer specific locale not supported
07a4:fixme:wbemprox:wbem_locator_ConnectServer unsupported flags
07a4:fixme:wbemprox:client_security_QueryBlanket semi-stub.
07a4:fixme:wbemprox:client_security_Release 74F41850
07a4:fixme:wbemprox:client_security_SetBlanket 74F41850, 09464440, 0, 0, L"<COLE_DEFAULT_PRINCIPAL>", 1, 3, 00000000, 0x20
07a4:fixme:wbemprox:client_security_Release 74F41850
07a4:fixme:wbemprox:wbem_services_QueryInterface interface {c3fcc19e-a970-11d2-8b5a-00a0c9b7c9c4} not implemented
07a4:fixme:wbemprox:wbem_services_QueryInterface interface {b196b283-bab4-101a-b69c-00aa00341d07} not implemented
07a4:fixme:wbemprox:wbem_services_QueryInterface interface {00000003-0000-0000-c000-000000000046} not implemented
07a4:fixme:wbemprox:wbem_services_QueryInterface interface {00000144-0000-0000-c000-000000000046} not implemented
07a4:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
07a4:fixme:wmiutils:wmiutils_cf_QueryInterface interface {b196b28f-bab4-101a-b69c-00aa00341d07} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {c3fcc19e-a970-11d2-8b5a-00a0c9b7c9c4} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {00000003-0000-0000-c000-000000000046} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {00000144-0000-0000-c000-000000000046} not implemented
07a4:fixme:wbemprox:wbem_services_CreateInstanceEnum unsupported flags 0x11
07a4:fixme:wbemprox:client_security_QueryBlanket semi-stub.
07a4:fixme:wbemprox:client_security_Release 74F41850
07a4:fixme:wbemprox:client_security_SetBlanket 74F41850, 093F0A68, 0, 0, L"<COLE_DEFAULT_PRINCIPAL>", 1, 3, 00000000, 0x20
07a4:fixme:wbemprox:client_security_Release 74F41850
07a4:fixme:wbemprox:enum_class_object_QueryInterface interface {c3fcc19e-a970-11d2-8b5a-00a0c9b7c9c4} not implemented
07a4:fixme:wbemprox:enum_class_object_QueryInterface interface {b196b283-bab4-101a-b69c-00aa00341d07} not implemented
07a4:fixme:wbemprox:enum_class_object_QueryInterface interface {00000003-0000-0000-c000-000000000046} not implemented
07a4:fixme:wbemprox:enum_class_object_QueryInterface interface {00000144-0000-0000-c000-000000000046} not implemented
07a4:fixme:wbemprox:client_security_QueryBlanket semi-stub.
07a4:fixme:wbemprox:client_security_Release 74F41850
07a4:fixme:wbemprox:client_security_SetBlanket 74F41850, 093F0A88, 0, 0, L"<COLE_DEFAULT_PRINCIPAL>", 1, 3, 00000000, 0x20
07a4:fixme:wbemprox:client_security_Release 74F41850
07a4:fixme:wbemprox:enum_class_object_QueryInterface interface {c3fcc19e-a970-11d2-8b5a-00a0c9b7c9c4} not implemented
07a4:fixme:wbemprox:enum_class_object_QueryInterface interface {b196b283-bab4-101a-b69c-00aa00341d07} not implemented
07a4:fixme:wbemprox:enum_class_object_QueryInterface interface {00000003-0000-0000-c000-000000000046} not implemented
07a4:fixme:wbemprox:enum_class_object_QueryInterface interface {00000144-0000-0000-c000-000000000046} not implemented
07a4:fixme:wbemprox:class_object_QueryInterface interface {c3fcc19e-a970-11d2-8b5a-00a0c9b7c9c4} not implemented
07a4:fixme:wbemprox:class_object_QueryInterface interface {b196b283-bab4-101a-b69c-00aa00341d07} not implemented
07a4:fixme:wbemprox:class_object_QueryInterface interface {00000003-0000-0000-c000-000000000046} not implemented
07a4:fixme:wbemprox:class_object_QueryInterface interface {00000144-0000-0000-c000-000000000046} not implemented
07a4:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
07a4:fixme:wmiutils:wmiutils_cf_QueryInterface interface {b196b28f-bab4-101a-b69c-00aa00341d07} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {c3fcc19e-a970-11d2-8b5a-00a0c9b7c9c4} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {00000003-0000-0000-c000-000000000046} not implemented
07a4:fixme:wmiutils:path_QueryInterface interface {00000144-0000-0000-c000-000000000046} not implemented
07a4:fixme:wmiutils:wmiutils_cf_QueryInterface interface {b196b28f-bab4-101a-b69c-00aa00341d07} not implemented
07a4:fixme:wmiutils:status_code_QueryInterface interface {c3fcc19e-a970-11d2-8b5a-00a0c9b7c9c4} not implemented
07a4:fixme:wmiutils:status_code_QueryInterface interface {00000003-0000-0000-c000-000000000046} not implemented
07a4:fixme:wmiutils:status_code_QueryInterface interface {00000144-0000-0000-c000-000000000046} not implemented
07a4:fixme:wmiutils:status_code_GetErrorCodeText 093F0B68, 0x80041002, 0, 0x1, 07C2F1F8
07fc:fixme:wincodecs:jpeg_decoder_get_metadata_blocks stub
02f8:fixme:ole:thread_context_callback_ContextCallback 093EF2F4, 78FC8272, 039CFBD8, {d7174f82-36b8-4aa8-800a-e963ab2dfab9}, 2, 00000000
02f8:fixme:ole:thread_context_callback_ContextCallback 093EF2F4, 78FC8272, 039CFB64, {d7174f82-36b8-4aa8-800a-e963ab2dfab9}, 2, 00000000
02f8:fixme:ole:thread_context_callback_ContextCallback 093EF2F4, 78FC8272, 039CFB64, {d7174f82-36b8-4aa8-800a-e963ab2dfab9}, 2, 00000000
02f8:fixme:ole:thread_context_callback_ContextCallback 093EF2F4, 78FC8272, 039CFB64, {d7174f82-36b8-4aa8-800a-e963ab2dfab9}, 2, 00000000
02f8:fixme:ole:thread_context_callback_ContextCallback 093EF2F4, 78FC8272, 039CF9E0, {d7174f82-36b8-4aa8-800a-e963ab2dfab9}, 2, 00000000
07fc:fixme:winspool:PerfOpen (null): stub
07fc:fixme:winspool:PerfCollect L"Global", 0852E304, 0852E2F8, 0852E2FC: stub
07fc:fixme:winspool:PerfClose stub
02ec:fixme:process:SetProcessPriorityBoost (00000698,0): stub
07fc:fixme:winspool:PerfOpen (null): stub
07fc:fixme:winspool:PerfCollect L"Global", 0852E304, 0852E2F8, 0852E2FC: stub
07fc:fixme:winspool:PerfClose stub
07fc:fixme:winspool:PerfOpen (null): stub
07fc:fixme:winspool:PerfCollect L"Global", 0852E304, 0852E2F8, 0852E2FC: stub
07fc:fixme:winspool:PerfClose stub
02ec:fixme:advapi:RegisterEventSourceW ((null),L".NET Runtime"): stub
02ec:fixme:advapi:ReportEventW (CAFE4242,0x0001,0x0000,0x00000402,00000000,0x0001,0x00000000,0094D360,00000000): stub
02ec:err:eventlog:ReportEventW L"Application: osu!.exe\n"
02ec:err:eventlog:ReportEventW L"Framework Version: v4.0.30319\n"
02ec:err:eventlog:ReportEventW L"Description: The process was terminated due to an unhandled exception.\n"
02ec:err:eventlog:ReportEventW L"Exception Info: System.AccessViolationException\n"
02ec:err:eventlog:ReportEventW L"Stack:\n"
02ec:err:eventlog:ReportEventW L"   at System.Drawing.SafeNativeMethods+Gdip.GdipMeasureString(System.Runtime.InteropServices.HandleRef, System.String, Int32, System.Runtime.InteropServices.HandleRef, System.Drawing.Internal.GPRECTF ByRef, System.Runtime.InteropServices.HandleRef, System.Drawing.Internal.GPRECTF ByRef, I"...
02ec:err:eventlog:ReportEventW L"   at System.Drawing.Graphics.MeasureString(System.String, System.Drawing.Font, System.Drawing.SizeF, System.Drawing.StringFormat)\n"
02ec:err:eventlog:ReportEventW L"   at System.Drawing.Graphics.MeasureString(System.String, System.Drawing.Font)\n"
02ec:err:eventlog:ReportEventW L"   at #=zrC6fmxS1WxFVMqvpHXvQu3ZwqB73x8zhLg==.#=zjXgYHWU=(System.String, Single, Microsoft.Xna.Framework.Vector2, Microsoft.Xna.Framework.Graphics.Color, #=zUj4WOKyr$zOG6FcNdtHWKTnbqE1hyqOLlg==, Boolean, Boolean, Boolean, #=zPqH6zaCEGNnWFnNRghOhAG1TNWIXcNiuyw==, Boolean, Microsoft.Xna.Fra"...
02ec:err:eventlog:ReportEventW L"   at #=zaw3v4oyoK1UurAxTs6DpjX2hZjyb.#=zNcFckSJgsC_O()\n"
02ec:err:eventlog:ReportEventW L"   at #=zaw3v4oyoK1UurAxTs6DpjX2hZjyb.#=zXfvnxtE=()\n"
02ec:err:eventlog:ReportEventW L"   at #=zD_QeDB_30LZFv9sPx9OrO2nYdHChutiU$w==.#=z$N47dFs=()\n"
02ec:err:eventlog:ReportEventW L"   at #=z6lARKPahGdYJDJSWxBnvvDrWvBrZzMgC2w==.#=z$N47dFs=()\n"
02ec:err:eventlog:ReportEventW L"   at #=zSolLQE8zU0hNdSxYlg==.#=z$N47dFs=()\n"
02ec:err:eventlog:ReportEventW L"   at #=zbBgu9jAq047lIciG3g==.#=zgwRV4$leeTx2(System.Object, System.EventArgs)\n"
02ec:err:eventlog:ReportEventW L"   at #=z9HvKcVfsL_flubi8yq0DRds=.#=zydgUSJM=(System.Object, System.EventArgs)\n"
02ec:err:eventlog:ReportEventW L"   at #=zBS24lRlbfZONCFP7Trv0V5s=.#=zicS1MwRYrMffjxHsaQ==(System.Object, System.EventArgs)\n"
02ec:err:eventlog:ReportEventW L"   at System.Windows.Forms.Application+ThreadContext.System.Windows.Forms.UnsafeNativeMethods.IMsoComponent.FDoIdle(Int32)\n"
02ec:err:eventlog:ReportEventW L"   at System.Windows.Forms.Application+ComponentManager.System.Windows.Forms.UnsafeNativeMethods.IMsoComponentManager.FPushMessageLoop(IntPtr, Int32, Int32)\n"
02ec:err:eventlog:ReportEventW L"   at System.Windows.Forms.Application+ThreadContext.RunMessageLoopInner(Int32, System.Windows.Forms.ApplicationContext)\n"
02ec:err:eventlog:ReportEventW L"   at System.Windows.Forms.Application+ThreadContext.RunMessageLoop(Int32, System.Windows.Forms.ApplicationContext)\n"
02ec:err:eventlog:ReportEventW L"   at System.Windows.Forms.Application.Run(System.Windows.Forms.Form)\n"
02ec:err:eventlog:ReportEventW L"   at #=zBS24lRlbfZONCFP7Trv0V5s=.#=z0gndKpc=()\n"
02ec:err:eventlog:ReportEventW L"   at #=zbBgu9jAq047lIciG3g==.#=z0gndKpc=()\n"
02ec:err:eventlog:ReportEventW L"   at #=zDEp43kp$i1e8dyW1GA==.#=zof7v5VAQXr7m(System.String, Boolean, Boolean)\n"
02ec:err:eventlog:ReportEventW L"   at #=zDEp43kp$i1e8dyW1GA==.#=zNVqmMtI=(System.String[])\n"
02ec:err:eventlog:ReportEventW L"   at #=zDEp43kp$i1e8dyW1GA==.#=zRDrqHZg=()\n"
02ec:err:eventlog:ReportEventW L"   at #=zunel3SWeRUiPuL_z9H9RFjdqrava.#=zM4atwFE=()\n"
02ec:fixme:advapi:DeregisterEventSource (CAFE4242) stub

Unhandled Exception: System.AccessViolationException: Attempted to read or write protected memory. This is often an indication that other memory is corrupt.
   at System.Drawing.SafeNativeMethods.Gdip.GdipMeasureString(HandleRef graphics, String textString, Int32 length, HandleRef font, GPRECTF& layoutRect, HandleRef stringFormat, GPRECTF& boundingBox, Int32& codepointsFitted, Int32& linesFilled)
   at System.Drawing.Graphics.MeasureString(String text, Font font, SizeF layoutArea, StringFormat stringFormat)
   at System.Drawing.Graphics.MeasureString(String text, Font font)
   at #=zrC6fmxS1WxFVMqvpHXvQu3ZwqB73x8zhLg==.#=zjXgYHWU=(String #=zjvdlhts=, Single #=zaj6nN98=, Vector2 #=zCe4cPBPiKohtbGJrBA==, Color #=zKkpmchI=, #=zUj4WOKyr$zOG6FcNdtHWKTnbqE1hyqOLlg== #=zsgfwiuw=, Boolean #=zZeBUzL1DlcmW, Boolean #=zqeQz7sVBNkfG, Boolean #=ztQO8rUY0k6Jcys3n0w==, #=zPqH6zaCEGNnWFnNRghOhAG1TNWIXcNiuyw== #=ztpeHPYQ=, Boolean #=zKn6vie4q5bAF, Vector2& #=zebQT7yHvvIZz2UhJVg==, RectangleF[]& #=zlFdS7oIcZ8Bi, Color #=zeena7aU=, Color #=zkpDg4o0=, Int32 #=zCd1B1Fvwrqgf, Boolean #=zO0m8YUanRx6W, Boolean #=zuVj5g9qjWsdi, #=zqIR67gA_TX5Yt6$W3YsQxRNUllfICLQVWw== #=zXCA5bwuHTxvr, Vector4 #=zdHpZwOqpbQXc, Vector2 #=zXSZwq4Y=, #=z0dOeMofo6XvxeqaGVg== #=zbZtfPoyigONv, Int32 #=zZvo0WPo=, Int32 #=zG99u9vk=)
   at #=zaw3v4oyoK1UurAxTs6DpjX2hZjyb.#=zNcFckSJgsC_O()
   at #=zaw3v4oyoK1UurAxTs6DpjX2hZjyb.#=zXfvnxtE=()
   at #=zD_QeDB_30LZFv9sPx9OrO2nYdHChutiU$w==.#=z$N47dFs=()
   at #=z6lARKPahGdYJDJSWxBnvvDrWvBrZzMgC2w==.#=z$N47dFs=()
   at #=zSolLQE8zU0hNdSxYlg==.#=z$N47dFs=()
   at #=zbBgu9jAq047lIciG3g==.#=zgwRV4$leeTx2(Object #=zTYbifUY=, EventArgs #=zs7ELP1A=)
   at #=z9HvKcVfsL_flubi8yq0DRds=.#=zydgUSJM=(Object #=zTYbifUY=, EventArgs #=zX_xQqts=)
   at #=zBS24lRlbfZONCFP7Trv0V5s=.#=zicS1MwRYrMffjxHsaQ==(Object #=zTYbifUY=, EventArgs #=zs7ELP1A=)
   at System.Windows.Forms.Application.ThreadContext.System.Windows.Forms.UnsafeNativeMethods.IMsoComponent.FDoIdle(Int32 grfidlef)
   at System.Windows.Forms.Application.ComponentManager.System.Windows.Forms.UnsafeNativeMethods.IMsoComponentManager.FPushMessageLoop(IntPtr dwComponentID, Int32 reason, Int32 pvLoopData)
   at System.Windows.Forms.Application.ThreadContext.RunMessageLoopInner(Int32 reason, ApplicationContext context)
   at System.Windows.Forms.Application.ThreadContext.RunMessageLoop(Int32 reason, ApplicationContext context)
   at System.Windows.Forms.Application.Run(Form mainForm)
   at #=zBS24lRlbfZONCFP7Trv0V5s=.#=z0gndKpc=()
   at #=zbBgu9jAq047lIciG3g==.#=z0gndKpc=()
   at #=zDEp43kp$i1e8dyW1GA==.#=zof7v5VAQXr7m(String #=zp$shfrMveHZY, Boolean #=zXn7bx_liBIzWBI6CPuzsds8=, Boolean #=zLztQdHnNL6urz7wW2Q==)
   at #=zDEp43kp$i1e8dyW1GA==.#=zNVqmMtI=(String[] #=zX_xQqts=)
   at #=zDEp43kp$i1e8dyW1GA==.#=zRDrqHZg=()
   at #=zunel3SWeRUiPuL_z9H9RFjdqrava.#=zM4atwFE=()
0424:fixme:kernelbase:AppPolicyGetThreadInitializationType FFFFFFFA, 15B4FF18
02ec:fixme:secur32:get_cipher_algid unknown algorithm 23
02ec:fixme:secur32:get_mac_algid unknown algorithm 200, cipher 23
02ec:fixme:secur32:get_cipher_algid unknown algorithm 23
02ec:fixme:secur32:get_mac_algid unknown algorithm 200, cipher 23
02ec:fixme:msvcrt:__clean_type_info_names_internal (60086D34) stub

**Comment by TextAdventurer12** [comment]
why are you using wine? lazer has native linux support

**Reply by 424ever** [reply]
1. You can see from the path that it is stable: `/A[ppData]/Local/osu!/osu!.exe`, which is where stable installs itself, as opposed to `AppData\Local\osu!lazer\osu!.exe`
2. I'm not even sure wine works with .NET

**Reply by rXelelo** [reply]
because i want play normat osu! i love it

**Comment by 424ever** [comment]
This is not the repository to report issues with osu! stable. That would be [here](https://github.com/ppy/osu-stable-issues). Or, you can check out the forums in search for help.
